ksonnet-gen/main.go

```diff
@@ -1,13 +1,13 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
 	"io/ioutil"
 	"log"
 	"os"
 
-	k8s "github.com/ksonnet/ksonnet-lib/ksonnet-gen/k8sSwagger"
-	"github.com/ksonnet/ksonnet-lib/ksonnet-gen/ksonnet"
+	"github.com/ksonnet/ksonnet-lib/ksonnet-gen/swagger"
```

Comment:
I'd consider using https://github.com/go-openapi/loads instead of doing your own serialization.